opsdroid/connector/telegram/__init__.py

```diff
@@ -212,13 +211,13 @@ def build_url(self, method):
                 _(
                     "Can't get updates because previous "
```

Comment:
Sorry this one is even uglier than the other ones haha
this string should be:
`_("Can't get updates because previous webhook is still active. Will try to delete webhook.")`